opentelemetry-tracing/opentelemetry-dotnet/log-trace-correlation/README.md

```diff
@@ -0,0 +1,128 @@
+# Splunk Distribution of OpenTelemetry .NET log trace correlation examples
+
+This folder contains sample applications instrumented using
+[Splunk Distribution of OpenTelemetry .NET](https://github.com/signalfx/splunk-otel-dotnet)
+that produce logs using logging API from `Microsoft.Extensions.Logging` enriched with
+a trace context.
+
+## Building and running the example setup
+
+This multi-container application assumes you have [Docker Compose](https://docs.docker.com/compose/) installed on your system.
+To build and run the example services, clone this repository, move to this folder and from there do the following:
+
+```sh
+docker-compose build
+SPLUNK_ACCESS_TOKEN=<access_token> SPLUNK_REALM=<realm> SPLUNK_HEC_URL=<splunk_hec_url> SPLUNK_HEC_TOKEN=<splunk_hec_token> docker-compose up
+```
+
+The value for `SPLUNK_ACCESS_TOKEN` can be found
+[here](https://app.signalfx.com/o11y/#/organization/current?selectedKeyValue=sf_section:accesstokens).
+Reference: [docs](https://docs.splunk.com/Observability/admin/authentication-tokens/api-access-tokens.html#admin-api-access-tokens).
+
+The value for `SPLUNK_REALM` can be found
+[here](https://app.signalfx.com/o11y/#/myprofile).
+Reference: [docs](https://docs.splunk.com/Observability/admin/allow-services.html).
+
+See [Splunk documentation](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector) to learn how to obtain
+values for `SPLUNK_HEC_URL` and `SPLUNK_HEC_TOKEN`.
+
+## Automatic correlation
```

Comment:
In future PR it would be good to add mention that:
1. `Example.LogTraceCorrelation.Console` is an example console application using `ILogger` with additional manual correlation for `Serilog`
2. `Example.LogTraceCorrelation.Web` is an example ASP.NET Core application using `ILogger` with additional manual correlation for `NLog`